### PR TITLE
chore(deps): update to 2021 version and update dependencies

### PR DESCRIPTION
**Pull request by flosse**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -2,7 +2,7 @@ mod server;
 
 use clap::{App, Arg};
 
-#[tokio::main]
+#[tokio::main(flavor = "current_thread")]
```
Comment (robertohuertasm) [inline src/main.rs]: why not the default `multi_thread`?

Comment (flosse) [reply src/main.rs]: Actually I forgot my motivation :see_no_evil: :rofl: 
I updated this PR now.

**Review by robertohuertasm — APPROVED** [review]
LGTM 🚀

**Comment by robertohuertasm** [comment]
Thank you @flosse for your PR. It's already published as 2.1.

**Pull request merged**